mlir/lib/Transforms/Utils/DialectConversion.cpp

```diff
@@ -820,6 +740,238 @@ void ArgConverter::insertConversion(Block *newBlock,
   conversionInfo.insert({newBlock, std::move(info)});
 }
 
+//===----------------------------------------------------------------------===//
+// RewriteAction
```

Comment:
Mentioned it in the other PR: do we have an alternative to "Action" here?

Reply:
I renamed `RewriteAction` to `IRRewrite`.